kogito-codegen-modules/kogito-codegen-core/src/main/java/org/kie/kogito/codegen/core/di/SpringDependencyInjectionAnnotator.java

```diff
@@ -169,4 +172,16 @@ public String emitterType(String dataType) {
                 .map(node::getAnnotationByName)
                 .anyMatch(Optional::isPresent);
     }
+
+    @Override
+    public Optional<String> getEndpointValue(MethodDeclaration md) {
+        Optional<AnnotationExpr> path = md.getAnnotationByName("PostMapping");
+        if (path.isPresent()) {
+            Optional<MemberValuePair> value = path.get().asNormalAnnotationExpr().getPairs().stream().filter(x -> x.getName().asString().equals("value")).findFirst();
+            if (value.isPresent()) {
+                return Optional.of(value.get().getValue().asStringLiteralExpr().asString());
+            }
+        }
+        return Optional.empty();
```

Comment:
This should be equivalent :thinking: 
```suggestion
        Optional<AnnotationExpr> path = md.getAnnotationByName("PostMapping");
        return path.flatMap(p -> p.asNormalAnnotationExpr().getPairs().stream()
            .filter(x -> "value".equals(x.getName().asString()).findFirst())
        )
        .map(value -> value.getValue().asStringLiteralExpr().asString());
```